geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/security/SimpleFeatureSecurityWrapper.scala

```diff
@@ -0,0 +1,33 @@
+package org.locationtech.geomesa.utils.security
+
+import org.opengis.feature.simple.SimpleFeature
+
+import scala.language.implicitConversions
+
+/**
+ * Wraps a [[SimpleFeature]] allowing visibility to be accessed and modified.
+ * 
+ * Created by mmatz on 4/3/15.
+ */
+class SimpleFeatureSecurityWrapper(val sf: SimpleFeature) {
```

Comment:
You can use an implicit class instead of the Class+Object method:

```scala
implicit class SecurityWrappedSimpleFeature(val sf: SimpleFeature) extends SimpleFeature {
   def getViz = SecurityUtils.getViz(sf)
}```

Then to use, just import it:

```scala
import org.locationtech....SecurityWrappedSimpleFeature
val sf = ....
val viz = sf.getViz
```